java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_3/Rectangle.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture3.tasks._3;
+
+import java.util.Objects;
+
+public non-sealed class Rectangle implements Shape {
+    int firstSide;
```

Comment:
same: you need to use private fields to enforce encapsulation

Reply:
done